src/components/NcAppSidebar/NcAppSidebarTabs.vue

```diff
@@ -314,7 +314,23 @@ export default {
 	}
 }
 
-:deep(.checkbox-radio-switch--button-variant.checkbox-radio-switch) {
-	border: unset;
+// Override checkbox-radio-switch styles so that it looks like tabs
+:deep(.checkbox-radio-switch--button-variant) {
```

Comment:
This completely changes the UI of the component, should we instead change that component?
Because this is now very likely to break on slight changes of the internals of NcCheckboxRadioSwitch.
For the same reason I feel quite uncomfortable with that the many `!important` statements.

Reply:
Could we first learn what the intended goal is here? Do we intend to go back with the design completely? Is it just about small fixes?

Reply:
That said, I think we're trying to do way too many things with `NcCheckboxRadioSwitch`

Reply:
@marcoambrosini can you make sure it doesn’t break in these 2 places:
- Sharing flow where we use the component vertically (right?)
- Forms on top right, where we use it for View/Edit/Results

Reply:
> Could we first learn what the intended goal is here? Do we intend to go back with the design completely? Is it just about small fixes?

Yes to both: It is about small fixes, and the active tab being way too present has been identified as a visual issue. Going to a middle ground between the old and new one seems a good way to go.

Reply:
> Could we first learn what the intended goal is here? Do we intend to go back with the design completely? Is it just about small fixes?

@raimund-schluessler Yes to both: It is about small fixes, and the active tab being way too present has been identified as a visual issue. Going to a middle ground between the old and new one seems a good way to go.

Reply:
> Are we sure this won't cause a11y issues? Cc @JuliaKirschenheuter

It only changes styles, so the only possible a11y issues could be about **state styles** and **contrast**. With the contrast everything is fine.

But with states it is not.

The current tab should have hover and active state styles.

Before | After
---|---
![with-state](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/17964d26-540d-4c76-a332-b342f32341c2) | ![no-state](https://github.com/nextcloud-libraries/nextcloud-vue/assets/25978914/e916d42c-4474-4589-959a-07230a0d5926)